finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4DispatcherBase.scala

```diff
@@ -17,10 +20,15 @@ trait Netty4DispatcherBase[SendMsg <: Message, RecvMsg <: Message] {
   protected[this] def log: Logger
   protected[this] def prefix: String
   protected[this] def stats: StatsReceiver
+  protected[this] def failureThreshold: Option[FailureDetector.Config] = None
 
   protected[this] def transport: Transport[Http2Frame, Http2Frame]
-  protected[this] lazy val writer: H2Transport.Writer = Netty4H2Writer(transport)
+  protected[this] def failureDetector: FailureDetector = failureThreshold match {
+    case None => FailureDetector(NullConfig, () => Future.never, stats)
```

Comment:
the second two arguments are ignored when `NullConfig` is passed in.  Consider passing `null` for them so that their values aren't red herrings.